backend/src/services/project/project-service.ts

```diff
@@ -306,8 +309,19 @@ export const projectServiceFactory = ({
     return deletedProject;
   };
 
-  const getProjects = async (actorId: string) => {
+  const getProjects = async (actorId: string, populateOrgName?: boolean) => {
     const workspaces = await projectDAL.findAllProjects(actorId);
+    if (populateOrgName) {
+      const orgs = await orgDAL.findAllOrgsByUserId(actorId);
+      return workspaces.map((workspace) => {
+        const orgName = orgs.find((org) => org.id === workspace.orgId)?.name || "";
+        return {
+          ...workspace,
+          orgName,
+          displayName: `${workspace.name} (${orgName})`
```

Comment:
The response of the API should be generic but in this case it is templated specifically for CLI. We should try to make the API responses be generic so that it can be useful for many other cases not just for the CLI. Here is what i think we should do:

When users `init` using the CLI, it should ask them to select a org and then ask them to choose a project scoped from that org. Currenly in the cli, you will see all projects from all orgs as a long list but by first scoping by org it will be less overwhelming for users. What do you think?

Reply:
We can do that, and in that case we can keep the api same as it is was previously, i'll just change the cli code

Reply:
Awesome